README.md

```diff
@@ -132,8 +140,9 @@ The leak tracker will catch leaks only for instrumented objects (See [concepts](
 
 However, the good news is:
 
-1. Most of disposable Flutter Framework classes are already instrumented. So, Widget related
-leaks in your Flutter application will be caught without additional effort.
+1. Most of disposable Flutter Framework classes are already instrumented. So, if
+your Flutter application manages widgets in a way that creates leaks, the leaks
+will be caught without additional effort.
```

Comment:
**issue:** This reads with some difficulty. I would recommend rephrasing somewhat.
```suggestion
1. Most disposable Flutter Framework classes include instrumentation.
If how your Flutter app manages widgets results in leaks, Flutter will catch them.
```